src/Hydra/Serializer/DocumentationNormalizer.php

```diff
@@ -396,13 +396,13 @@ private function getClasses(array $entrypointProperties, array $classes): array
                 [
                     '@type' => 'hydra:SupportedProperty',
                     'hydra:property' => [
-                        '@id' => '#ConstraintViolationList/violation',
-                        '@type' => 'rdf:Property',
-                        'rdfs:label' => 'violation',
+                        '@id' => '#ConstraintViolationList/violations',
+                        '@type' => 'hydra:Link',
```

Comment:
@dunglas I looked for more informations but I found nothing. May be I am wrong but I don't know how to process a property as relation if `@type` is not `hydra:Link`